Source/WebCore/rendering/RenderLayerScrollableArea.cpp

```diff
@@ -845,13 +845,19 @@ bool RenderLayerScrollableArea::canShowNonOverlayScrollbars() const
     return canHaveScrollbars() && !(m_layer.renderBox() && m_layer.renderBox()->canUseOverlayScrollbars());
 }
 
+void RenderLayerScrollableArea::updateScrollbarsControllerAfterLayerCreation()
```

Comment:
This could be called `updateScrollbarsControllerIfNecessary()` since it's not specifically after layers have been newly created.

Reply:
Fixed.